webapp/utils/post_utils.jsx

```diff
@@ -30,3 +30,32 @@ export function getProfilePicSrcForPost(post, timestamp) {
 
     return src;
 }
+
+export function messageHistoryHandler(e) {
+    if ((e.ctrlKey || e.metaKey) && !e.altKey && !e.shiftKey && (e.keyCode === Constants.KeyCodes.UP || e.keyCode === Constants.KeyCodes.DOWN)) {
+        if (this.state.messageText !== '' && this.currentLastMsgIndex === PostStore.getHistoryLength()) {
```

Comment:
I suggested doing it like this since it's simple and is similar to how [mixins](http://egorsmirnov.me/2015/09/30/react-and-es6-part4.html) work with ES6 but I do concede it's confusing in the post_utils.jsx file so moving it somewhere else might be a good idea. How about we talk about it in the dev meeting?